model/ddl.go

```diff
@@ -429,4 +429,9 @@ type SchemaDiff struct {
 	OldTableID int64 `json:"old_table_id"`
 	// OldSchemaID is the schema ID before rename table, only used by rename table DDL.
 	OldSchemaID int64 `json:"old_schema_id"`
+
+	// AffectedTableID is the table ID of partition table when exchanging partition
+	AffectedTableID int64 `json:"affected_table_id"`
```

Comment:
Model data is persistented, so new fields of the struct is hard to remove for compatibility reasons once introduced.

Can we reuse `OldTableID` `OldSchemID` when the ActionType is AlterTableExchangePartition ?

Reply:
In theory, the fields are not enough  to use . For exchanging partition, the IDs of partition and `nt` are exchanged. nt need to clear the old table id , so it used `OldTableID` and `OldSchemaID`. Meanwhile there is no field to record the schemID and TableID of pt, it needs to expand.

Reply:
> Meanwhile there is no field to record the schemID and TableID of pt, it needs to expand.

Could we use `SchemaID` and `TableID` to record the schemID and TableID of `pt`?

Reply:
PTAL @tiancaiamao @zimulala